### PR TITLE
Release v0.23.15

### PR DESCRIPTION
**Pull request by armanbilge**

### CODE REVIEW

**Comment by bplommer** [comment]
Should we revert https://github.com/http4s/http4s/pull/6628 before this release, since there seems to be some disagreement over it?

**Comment by ChristopherDavenport** [comment]
I prefer that API, but if there is disagreement we can always hold off temporarily.

**Comment by danicheg** [comment]
I apologize for my grumpiness. I didn't want to block this release, and I also like suggested changes in type definitions of Logger helpers. It just has seemed to me that we moved slightly quickly without having discussed it well. So now it definitely seems that there is a quorum, so we are good to move onward. Sorry once again.

**Comment by bplommer** [comment]
No apology needed @danicheg - it was entirely my fault for being far too trigger-happy with the merge!

**Review by ChristopherDavenport — APPROVED** [review]

File: docs/changelog.md [hunk 1]
```diff
@@ -1,9 +1,74 @@
...
+* Don't backtrack in request/response prelude parsing by @TimWSpence in https://github.com/http4s/http4s/pull/6578
+
+### http4s-ember-server
+
+* Always respond to client close frame with 1000 "normal closure" by @yurique in https://github.com/http4s/http4s/pull/6594
+* Parse all WebSocket frames in a `Chunk` in ember-server by @buntec in https://github.com/http4s/http4s/pull/6587
+
+### Documentation
+
+* Release v0.23.14 by @armanbilge in https://github.com/http4s/http4s/pull/6568
```
Comment (danicheg) [inline docs/changelog.md:44]:
```suggestion
```

**Review by danicheg — APPROVED** [review]

**Review by isomarcte — APPROVED** [review]

**Comment by danicheg** [comment]
Sorry for adding a commit after that PR has been reviewed. Just don't want good work to get stuck waiting for a new release for a month.

**Pull request merged**